docs-css/docs-nav.css

```diff
@@ -0,0 +1,99 @@
+
+  /* header */
+  .navbar {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
```

Comment:
This code is repeated many times, you can make a util class for this and use it wherever you want it. It will reduce code duplicacy.

Reply:
will try to do so